plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlCockroachConnectorSmokeTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.postgresql;
+
+import io.trino.plugin.jdbc.BaseJdbcConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+
+import java.util.Map;
+
+import static io.trino.plugin.postgresql.PostgreSqlQueryRunner.createPostgreSqlQueryRunner;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+// TODO https://github.com/trinodb/trino/issues/13771 Move to CockroachDB connector
+public class TestPostgreSqlCockroachConnectorSmokeTest
```

Comment:
CockroachDB != PostgreSQL and Cockroack doesn't make compatibility promises

from https://www.cockroachlabs.com/docs/stable/postgresql-compatibility.html

> CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the **_majority_** of PostgreSQL syntax. This means that existing applications built on PostgreSQL can **_often_** be **_migrated_** to CockroachDB without changing application code.

(emphasis mine)

I think we should rather have a dedicated CockroachDB connector